test/ml/test_readme_sample.py

```diff
@@ -38,9 +38,9 @@ def print(*args):
         from qiskit.aqua import QuantumInstance, aqua_globals
         from qiskit.aqua.algorithms import VQC
         from qiskit.aqua.components.optimizers import COBYLA
-        from qiskit.aqua.components.feature_maps import RawFeatureVector
         from qiskit.ml.datasets import wine
         from qiskit.circuit.library import TwoLocal
+        from qiskit.ml.circuit.library import RawFeatureVector
```

Comment:
Since this is intended to reflect identically the sample code from the readme  can you update the readme inline with this - or we do it nearer the release (that and this test) if we want to keep the readme more inline with the stable version. In which case I would let this test emit the deprecation warning so before release we should see it and correct it.

Reply:
We can keep the old version until the release, I'll open an issue to not forget about it.

Reply:
Done in f907529